### PR TITLE
Feature parity: subscriber attributes

### PR DESCRIPTION
**Pull request by aboedo**
Added missing subscriber attributes. 
Currently testing on device

### CODE REVIEW

File: RevenueCat/Scripts/Purchases.cs [hunk 1]
```diff
@@ -872,6 +872,42 @@ public void SetAirshipChannelID(string airshipChannelID)
         _wrapper.SetAirshipChannelID(airshipChannelID);
     }
 
+    /**
+     * <summary>
+     * Subscriber attribute associated with the CleverTap ID for the user.
```
Comment (tonidero) [inline RevenueCat/Scripts/Purchases.cs:877]: NItpick. Since this is a setter, this should be `Sets subscriber attribute...` I think? Same for the others.

**Review by tonidero — APPROVED** [review]
Looks good! After it's been tested :shipit:

**Comment by aboedo** [comment]
tested on device for iOS and Android, working on both

File: RevenueCat/Scripts/Purchases.cs (hunk 1 above)
Comment (aboedo) [reply RevenueCat/Scripts/Purchases.cs]: good call, I updated it for all subs attributes

**Pull request merged**